core/arch/arm/kernel/user_ta.c

```diff
@@ -688,6 +689,39 @@ static TEE_Result add_deps(struct user_ta_ctx *utc __unused,
 
 #endif
 
+#ifdef CFG_TA_ASLR
+static size_t aslr_offset(size_t min, size_t max)
+{
+	uint32_t rnd32 = 0;
+	size_t rnd = 0;
+
+	assert(min <= max);
+	if (max > min) {
+		if (crypto_rng_read(&rnd32, sizeof(rnd32)))
+			return 0;
```

Comment:
I think, it is better to return `min` there. Or `max` :) 

Also, it is worth to print some warning to debug console.

Reply:
I'll return `min` and add a warning.